### DISCUSSION TITLE
Virtual machine image questions

### DISCUSSION THREAD

**Discussion by dajohnson21** [ltsp/ltsp#327] [Q&A] [answered]
Hi, I have LTSP working by using the chrootless method and now I am kind of curious about the Virtual Machine method. I guess I am wondering if, for example, I could boot a windows virtual machine. If not, what are the OS requirements for using the virtual machine method?

Thanks,
David

**Comment by alkisg** [comment, marked as answer]
Hi David,

the OS requirements for the client image are the same whether you're using chrootless, chroot or VM. Currently only Debian-based distributions are supported (initramfs-tools), and more Linux distributions (e.g. rpm/dracut) could be supported in the future.

There's no support for Windows images, but you could netboot Linux and then run a Windows VM on top of Linux.

**Reply by dajohnson21** [reply]
Thanks for the answer. I will definitely explore WINE.
